### DISCUSSION TITLE
Heavy nodes sends data to manager Redis

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#4697] [Unsupported Versions] [answered]
Hello!
I am using distributed scheme with heavy nodes. While using nodes I've noticed that some events from heavy node #1 can be indexed by heavy node #2 and vice versa. I've looked further and found out that heavy nodes sends data to manager Redis just like in distributed scheme with forwarders and search nodes. Temporarily I've changed Logstash pipelines config (redis output and input) so every node indexed only it's logs.
My question is why heavy nodes sends data to manager? As I can see in SO architecture heavy nodes shouldn't do that but to process all data by itself. Can I override this behavior
Thank you!

**Comment by dougburks** [comment, marked as answer]
Thanks for letting us know about this @Sahale . We've published a hotfix that should fix the heavy node pipeline:
https://blog.securityonion.net/2021/07/security-onion-2360-heavy-node-hotfix.html
